tests/prefetch_related/tests.py

```diff
@@ -256,6 +259,81 @@ def test_m2m_then_reverse_one_to_one_object_ids(self):
         self.assertWhereContains(sql, self.author1.id)
 
 
+@mock.patch('django.db.connection.features.max_query_params', 50)
+class LargePrefetchSetTests(TestCase):
+    """
+    Prefetch queries using large "WHERE id IN (…)" clauses are performed in
+    batches on databases with a limited number of variables per query, such as
+    SQLite. To improve test performance, a smaller value of max_query_params
+    is mocked.
+    """
+    test_size = 101
```

Comment:
Maybe define a module constant, like `fake_max_query_param = 50` and use it in the `mock.patch` call and to define `TEST_SIZE=fake_max_query_param * 2 + 1`.

Also, I think 50 is too much, shouldn't 10 be enough?